core/src/expand.rs

```diff
@@ -160,6 +173,8 @@ fn expand_compile_time_bindings(
         let mut fragment_str = fragment_string.as_str();
 
         while let Some(start_of_binding) = fragment_str.find('{') {
+            // REVIEW: In which context exactly is `{{` used to escape braces?
+            // REVIEW: I just didn't know about this :D. Is this an sqlx thing?
```

Comment:
No, this is just for this macro.  Without a way to escape curly brackets you could not use PostgreSQL array literals like `{"foo", "bar"}`.  I didn't document it though because `{{"foo", "bar"}` looks really awkward, and I figure we need to think about it more before documenting it.  Alternatively I would also be okay with just removing it for now.

Reply:
Yeah. I think it might be good to remove this for now.
We probably need a better way of escaping, as this also doesn't handle nested arrays.

Reply:
Hmm, nested arrays should work fine since you can use the `{{` escape as many times as you want, but it would indeed look very confusing!  Though I also don't think that's necessarily as much of an issue since SQLx sadly doesn't support nested arrays anyway, but we really should have a less confusing syntax choice before any form of escaping is released.

Do you want to remove it, or should I do it in a separate commit?  :)